src/assets/i18n/en.json

```diff
@@ -1,18 +1,16 @@
 {
-    "en": {
-      "translation": {
-        "buttons": {
-          "queue": "Queue",
-          "stack": "Stack",
-          "enqueue": "Enqueue",
-          "dequeue": "Dequeue",
-          "back": "Back",
-          "push": "Push",
-          "pop": "Pop"
-        },
-        "headers": {
-          "hi": "HI"
-        }
-      }
+  "translation": {
+    "buttons": {
```

Comment:
Such pattern is not very scalable for the future. In feature you might need to reuse for example `Queue` for other components than button and it will lead to inconsistency. It's better to split naming by domains as well. For example "QUEUE": {"T": "Queue"} (T presents title). Than you can add some actions in it as well. {"QUEUE": {"ADD": " Add queue}}